### PR TITLE
feat(protocol-designer): onboarding animations for ot-2

### PR DESCRIPTION
**Pull request by jerader**
closes RQA-3677

# Overview

add onboarding animations for ot-2

## Test Plan and Hands on Testing

Test that the ot-2 and flex onboarding flows work with the animations

## Changelog

- add new animations for ot-2
- add branching logic depending on robot type

## Risk assessment

low

### CODE REVIEW

File: protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx [hunk 1]
```diff
@@ -37,6 +40,21 @@ interface WizardBodyProps {
   tooltipOnDisabled?: string
 }
 
+const OT2_GIFS: Record<number, string> = {
```
Comment (jerader) [inline protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx:43]: Note: post-release we will revisit optimizing the animations for ot-2 and flex. Right now its a bit weird that the flex animations are in webm format but ot-2 animations are in gif format. but it seems to work okay for the release so i think its fine that they have different formats.

File: protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx [hunk 2]
```diff
@@ -153,21 +172,33 @@ export function WizardBody(props: WizardBodyProps): JSX.Element {
...
+        {robotType === FLEX_ROBOT_TYPE || stepNumber === 1 ? (
+          <video
+            preload="auto"
+            css={css`
+              width: 100%;
+              height: 100%;
+              object-fit: cover;
+              border-radius: ${BORDERS.borderRadius16};
+            `}
+            autoPlay={true}
```
Comment (ncdiehl11) [inline protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx]:
```suggestion
            autoPlay
```

**Review by ncdiehl11 — APPROVED** [review]
love the animations! one nit

File: protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx [hunk 3]
```diff
@@ -153,21 +172,33 @@ export function WizardBody(props: WizardBodyProps): JSX.Element {
...
+              height: 100%;
+              object-fit: cover;
+              border-radius: ${BORDERS.borderRadius16};
+            `}
+            autoPlay
+            loop={false}
+            controls={false}
+            aria-label={`onboarding animation for page ${stepNumber}`}
+          >
+            <source src={asset ?? ''} type="video/mp4" />
```
Comment (koji) [inline protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx]: why mp4? I thought the type was webm.
converted from webm to mp4?

Comment (jerader) [reply protocol-designer/src/pages/CreateNewProtocolWizard/WizardBody.tsx]: oops they're definitely webm, i'll fix

**Pull request merged**